07-html.Rmd

```diff
@@ -408,3 +408,60 @@ In Section \@ref(latex-hardcore), we mentioned that if you feel the constraint o
+
+
+## Sharing HTML content on the web
+
+One appealing aspect of rendering R Markdown content to HTML files is that it is very easy to host these files on the internet and share them just as one shares any other website. This section briefly summarizes numerous options for sharing the HTML documents which you create. 
+
+### R-specific services
+
+RStudio PBC offers a number of services for publishing various types of content created in R Markdown to the internet. These services make it particularly easy to publish content by using the RStudio IDE or the **rsconnect** package.
+
+- [**RPubs**](https://rpubs.com/) enables free hosting of static single-file R Markdown content. It is easy to publish via the Publish button in the RStudio IDE or the `rsconnect::rpubsUpload()` function. Please see the "Getting Started" page (https://rpubs.com/about/getting-started) for more details.
+
+- [**shinyapps.io**](https://www.shinyapps.io/) allows for hosting dynamic content which requires a server running R. For example, one can host interactive R Markdown documents which use the `runtime: shiny` option to include Shiny components. shinyapps.io is an analog to RPubs for Shiny applications. Apps and interactive R Markdowns can be published using the push-button in the RStudio IDE or the `rsconnect::deployApp()` function. See the user guide (https://docs.rstudio.com/shinyapps.io/index.html) for more details. 
+
+- [**bookdown.org**](https://bookdown.org/home/about/) offers free hosting specifically for books written with the **bookdown** package. You may easily publish static output files of your book using the `bookdown::publish_book()` function. This platform is based on RStudio PBC's enterprise RStudio Connect product (see below).
+
+- [**RStudio Connect**](https://rstudio.com/products/connect/) is an enterprise product which organizations may run on their own servers. It can host a wide variety of content created in R (R Markdown documents, Shiny apps, APIs) in a secured environment with document-level access controls, viewership history, and more. Similar to the open-source RPubs and shinyapps.io offerings, content can be published to RStudio Connect using manual upload, the **rsconnect** package, or with Git-based deployment. 
+
+### Static website services
+
+In addition to R-specific services, you may host your HTML document on many freely available static website hosting services. These include:
+
+- [**GitHub Pages**](https://pages.github.com/) makes is particularly easy to publish markdown and HTML content straight from a GitHub repository. You may specify whether to host content from either your main branch's root, your main branch's `docs/` folder, or a specific `gh-pages` branch. You can publish new content simply by pushing new HTML files to your repository.
+
+- [**GitLab Pages**](https://docs.gitlab.com/ee/user/project/pages/) offers similar functionality to GitHub Pages for GitLab repositories. 
```

Comment:
It may be better to link to the community edition help page: https://docs.gitlab.com/ce/user/project/pages/

We could also add _in a simple way_ that this is more advanced and less easy because you have to use the included CI tool, gitlab-ci, to publish your website. But, how to to say that without talking about CI ? 🤔

Reply:
Yikes! Thanks for the good catch on `ee` -> `ce`. 

To avoid mentioning CI directly, I added information here that you need to write a YAML file to tell GitLab what to do and linked to an example of what that looks like for already rendered HTML. 

I know that both GitHub and GitLab can link to SSGs, but since this doesn't feel super relevant outside of blogdown, I am not mentioning that. Do you think that is already?

Reply:
Your updates are fine I think ! 👍